src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/StatusBar.android.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Android.OS;
+using Android.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class StatusBar
+	{
+		static partial void PlatformSetColor(Color color)
+		{
+			if (Build.VERSION.SdkInt < BuildVersionCodes.M)
+				return;
+
+			var activity = (FormsAppCompatActivity)ToolkitPlatform.Context.GetActivity();
+			activity.SetStatusBarColor(color.ToAndroid());
+		}
+
+		static partial void PlatformSetStyle(StatusBarStyle style)
+		{
+			if (Build.VERSION.SdkInt < BuildVersionCodes.M)
+				return;
+
+			switch (style)
+			{
+				case StatusBarStyle.DarkContent:
+					BarStyle.AddBarAppearanceFlag((StatusBarVisibility)SystemUiFlags.LightStatusBar);
+					break;
+				case StatusBarStyle.LightContent:
```

Comment:
```suggestion
```

Reply:
This is there for readability. It's not obvious, what values are covered by `default` case

Reply:
Any value that isn't `DarkContent`

Reply:
Yes, but that's not future proof. What if we add `Auto` value, for example? `LightContent` logic will always be used for `LightContent` and `Default` values, but not for anything else.

Reply:
If we need to add we can refactor this late, premature optimization isn't good